### ISSUE TITLE
Research National Levee Layer

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/riverscapes-tools#650]
As part of #645, @philipbaileynar included levees as an optional input into Anthropogenic Projects:

> Levees (@joewheaton mentioned something about a national Levee dataset)

The reason they have never been used is because nothing consistent or of any good quality exists. A recent paper by  Knox et al. (2022) sounded promising. The paper has some summary products, but it is supposedly based on a national  "Potential Levees" [dataset](https://www.hydroshare.org/resource/729c0aea00bb48d6b6814c147e4318c4/):
- Knox RL, Morrison RR, Wohl EE. 2022. A river ran through it: Floodplains as America’s newest relict landform. Science Advances 8 : eabo1082. DOI: [10.1126/sciadv.abo1082](https://doi.org/10.1126/sciadv.abo1082)

There are also some helpful concepts in the paper worth taking a closer look at for @jtgilbert  (may hold on to later for RCAT): 

![image](https://user-images.githubusercontent.com/9723905/205106247-7419a74d-3997-4d59-ae70-10aa2f7fd214.png)

Here, we need to see if this levee dataset is good enough for use.

**Comment by joewheaton** [comment]
I downloaded and explored the dataset. It is awful. 
![image](https://user-images.githubusercontent.com/9723905/205320935-58a73b26-ce6e-403f-9c1d-ce1619b51b26.png)

It appears to be an incoherent vector polygon derived presumably from the landcover raster somehow? It is NOT an actual mapping of real levees. I checked locations with known levees and locations with out levees and found it missing known levees frequently and incorrectly identifying spaces as levees. We DO NOT WANT TO USE THIS.

FYI @jtgilbert, @philipbaileynar and @lauren-herbine
